pkg/asset/quota/quota.go

```diff
@@ -157,24 +157,6 @@ func (a *PlatformQuotaCheck) Generate(dependencies asset.Parents) error {
 		if err != nil {
 			return errors.Wrap(err, "failed to create a new PISession")
 		}
-
-		transitGatewayEnabled := configpowervs.TransitGatewayEnabledZone(ic.Config.Platform.PowerVS.Zone)
-
-		if !transitGatewayEnabled {
-			err = bxCli.ValidateCloudConnectionInPowerVSRegion(context.TODO(), ic.Config.Platform.PowerVS.ServiceInstanceID)
-			if err != nil {
-				return fmt.Errorf("failed to meet the prerequisite for Cloud Connections: %w", err)
-			}
-		}
-
-		if !transitGatewayEnabled {
-			if ic.Config.Platform.PowerVS.PVSNetworkName == "" {
-				err = bxCli.ValidateDhcpService(context.TODO(), ic.Config.Platform.PowerVS.ServiceInstanceID, ic.Config.MachineNetwork)
```

Comment:
Should remove the ValidateDhcpService function from the client as well I think...

Reply:
Yeah, it is not used elsewhere!